### DISCUSSION TITLE
How to delete blog posts from Headless CMS ? i.e. how to remove data from .json or delete .md file.

### DISCUSSION THREAD

**Discussion by dnyand33p** [11ty/eleventy#1676] [Q&A] [answered]
I have created my first blog with eleventy.
For blog posts -> created a "posts" folder -> put a .md file for each post with front matter (and "post" tag).

Then in Headless CMS Admin Panel, I have "grabbed" collections.posts to display the list of posts.
Now I want to delete some posts from within the CMS itself.
How can I do that?

i.e. **How can we remove the data from .json data file** or remove the .md file or shall I remove items from collections at build time with some way of filtering?

[Link to repo](https://github.com/sampada-organization/self)

**Comment by Ryuno-Ki** [comment, marked as answer]
Which CMS did you use? (e.g. a link if you're unsure).
Eleventy itself is a static site generator. But it can be combined with CMS.

**Reply by dnyand33p** [reply]
I'm using Netlify CMS

**Reply by dnyand33p** [reply]
I have combined Eleventy with a Netlify CMS  so I can now add new posts, but not sure how to delete the posts - which are being generated from .md files (or .json). I'm fairly new to JAMStack and browsing docs but lack in big picture so...

**Reply by dnyand33p** [reply]
[Link to the repo: ](https://github.com/sampada-organization/self)

**Reply by Ryuno-Ki** [reply]
Hm, since the [Netlify CMS docs](https://www.netlifycms.org/docs/add-to-your-site/) don't say much about it, my wild guess would be to delete those files in GitHub.

They [link to Slack](https://netlifycms.org/chat) on their site (I assume, it's more a Netlify question than an Eleventy one right now).

**Reply by dnyand33p** [reply]
I generated blog posts from .md files so the Delete option is not showing in Netlify CMS, but if I use the data from .json file to generate posts then I can do so from within the CMS.
